hack/run-ci-e2e-test.sh

```diff
@@ -10,9 +10,56 @@ NODE_COUNT=""
 SKIP_NODE_DELETION=""
 KEY_PAIR_NAME=""
 
-
 export CGO_ENABLED=0
 
+get_WMCO_logs() {
+  oc logs -l name=windows-machine-config-operator -n windows-machine-config-operator
+}
+
+# This function runs operator-sdk run --olm/cleanup depending on the given parameters
+# Parameters:
+# 1: command to run [run/cleanup]
+# 2: path to the operator-sdk binary to use
+# 3: path to the directory holding the operator manifests
+OSDK_WMCO_management() {
+  if [ $# != 3 ]; then
+    echo incorrect parameter count for OSDK_WMCO_management
+    return 1
+  fi
+  if [[ "$1" != "run" && "$1" != "cleanup" ]]; then
+    echo $1 does not match either run or cleanup
+    return 1
+  fi
+
+  local COMMAND=$1
```

Comment:
Shouldn't this be before [line](https://github.com/openshift/windows-machine-config-operator/pull/61/files#diff-98085f3fd979233fe093e8f731f11240R29)